tests/Cookie/CookieJarTest.php

```diff
@@ -37,6 +37,11 @@ public function testCreatesFromArray()
         $this->assertCount(2, $jar);
     }
 
+    public function testEmptyJarIsCountable()
+    {
+        self::assertCount(0, new CookieJar());
```

Comment:
Having `count` method implemented does not guarantees that id does not fail. Short example:

```php
class foo implements \Countable {
   private $things;

   public function count() { return count($this->things); }

   public function push($element) { $this->things[] = $element; } 
}

$f = new \foo();
$f->count(); // this will pass on PHP < 7.2 and throw warning on PHP 7.2
$f->push(1);
$f->count(); // this will pass
```

So having `count` method implemented does not guarantee that it **works properly internally**.

Checking that `count == 0` is needed because the future phpunit versions will mark tests without assertions as risky, actually I could just call the `(new CookieJar())->count()` without any assertions.